From 8cdf905f40ba1bdabbc0fde8a6d1f501000f9b8d Mon Sep 17 00:00:00 2001
From: Hideki Yamane <henrich@debian.org>
Date: Sat, 24 Feb 2018 22:15:07 +0900
Subject: [PATCH 1/3] relocate distro scripts files to each distribution

Adding new distro/version to scripts directory makes it messy. Divided into
each distro is better.
---
 Makefile                              | 2 +-
 scripts/{ => debian}/bullseye         | 0
 scripts/{ => debian}/buster           | 0
 scripts/{ => debian}/etch             | 0
 scripts/{ => debian}/etch-m68k        | 0
 scripts/{ => debian}/jessie           | 0
 scripts/{ => debian}/jessie-kfreebsd  | 0
 scripts/{ => debian}/lenny            | 0
 scripts/{ => debian}/oldoldstable     | 0
 scripts/{ => debian}/oldstable        | 0
 scripts/{ => debian}/potato           | 0
 scripts/{ => debian}/sarge            | 0
 scripts/{ => debian}/sarge.buildd     | 0
 scripts/{ => debian}/sarge.fakechroot | 0
 scripts/{ => debian}/sid              | 0
 scripts/{ => debian}/squeeze          | 0
 scripts/{ => debian}/stable           | 0
 scripts/{ => debian}/stretch          | 0
 scripts/{ => debian}/testing          | 0
 scripts/{ => debian}/unstable         | 0
 scripts/{ => debian}/wheezy           | 0
 scripts/{ => debian}/woody            | 0
 scripts/{ => debian}/woody.buildd     | 0
 scripts/{ => tanglu}/aequorea         | 0
 scripts/{ => tanglu}/bartholomea      | 0
 scripts/{ => tanglu}/chromodoris      | 0
 scripts/{ => tanglu}/dasyatis         | 0
 scripts/{ => ubuntu}/artful           | 0
 scripts/{ => ubuntu}/bionic           | 0
 scripts/{ => ubuntu}/breezy           | 0
 scripts/{ => ubuntu}/dapper           | 0
 scripts/{ => ubuntu}/edgy             | 0
 scripts/{ => ubuntu}/feisty           | 0
 scripts/{ => ubuntu}/gutsy            | 0
 scripts/{ => ubuntu}/hardy            | 0
 scripts/{ => ubuntu}/hoary            | 0
 scripts/{ => ubuntu}/hoary.buildd     | 0
 scripts/{ => ubuntu}/intrepid         | 0
 scripts/{ => ubuntu}/jaunty           | 0
 scripts/{ => ubuntu}/karmic           | 0
 scripts/{ => ubuntu}/lucid            | 0
 scripts/{ => ubuntu}/maverick         | 0
 scripts/{ => ubuntu}/natty            | 0
 scripts/{ => ubuntu}/oneiric          | 0
 scripts/{ => ubuntu}/precise          | 0
 scripts/{ => ubuntu}/quantal          | 0
 scripts/{ => ubuntu}/raring           | 0
 scripts/{ => ubuntu}/saucy            | 0
 scripts/{ => ubuntu}/trusty           | 0
 scripts/{ => ubuntu}/utopic           | 0
 scripts/{ => ubuntu}/vivid            | 0
 scripts/{ => ubuntu}/warty            | 0
 scripts/{ => ubuntu}/warty.buildd     | 0
 scripts/{ => ubuntu}/wily             | 0
 scripts/{ => ubuntu}/xenial           | 0
 scripts/{ => ubuntu}/yakkety          | 0
 scripts/{ => ubuntu}/zesty            | 0
 57 files changed, 1 insertion(+), 1 deletion(-)
 rename scripts/{ => debian}/bullseye (100%)
 rename scripts/{ => debian}/buster (100%)
 rename scripts/{ => debian}/etch (100%)
 rename scripts/{ => debian}/etch-m68k (100%)
 rename scripts/{ => debian}/jessie (100%)
 rename scripts/{ => debian}/jessie-kfreebsd (100%)
 rename scripts/{ => debian}/lenny (100%)
 rename scripts/{ => debian}/oldoldstable (100%)
 rename scripts/{ => debian}/oldstable (100%)
 rename scripts/{ => debian}/potato (100%)
 rename scripts/{ => debian}/sarge (100%)
 rename scripts/{ => debian}/sarge.buildd (100%)
 rename scripts/{ => debian}/sarge.fakechroot (100%)
 rename scripts/{ => debian}/sid (100%)
 rename scripts/{ => debian}/squeeze (100%)
 rename scripts/{ => debian}/stable (100%)
 rename scripts/{ => debian}/stretch (100%)
 rename scripts/{ => debian}/testing (100%)
 rename scripts/{ => debian}/unstable (100%)
 rename scripts/{ => debian}/wheezy (100%)
 rename scripts/{ => debian}/woody (100%)
 rename scripts/{ => debian}/woody.buildd (100%)
 rename scripts/{ => tanglu}/aequorea (100%)
 rename scripts/{ => tanglu}/bartholomea (100%)
 rename scripts/{ => tanglu}/chromodoris (100%)
 rename scripts/{ => tanglu}/dasyatis (100%)
 rename scripts/{ => ubuntu}/artful (100%)
 rename scripts/{ => ubuntu}/bionic (100%)
 rename scripts/{ => ubuntu}/breezy (100%)
 rename scripts/{ => ubuntu}/dapper (100%)
 rename scripts/{ => ubuntu}/edgy (100%)
 rename scripts/{ => ubuntu}/feisty (100%)
 rename scripts/{ => ubuntu}/gutsy (100%)
 rename scripts/{ => ubuntu}/hardy (100%)
 rename scripts/{ => ubuntu}/hoary (100%)
 rename scripts/{ => ubuntu}/hoary.buildd (100%)
 rename scripts/{ => ubuntu}/intrepid (100%)
 rename scripts/{ => ubuntu}/jaunty (100%)
 rename scripts/{ => ubuntu}/karmic (100%)
 rename scripts/{ => ubuntu}/lucid (100%)
 rename scripts/{ => ubuntu}/maverick (100%)
 rename scripts/{ => ubuntu}/natty (100%)
 rename scripts/{ => ubuntu}/oneiric (100%)
 rename scripts/{ => ubuntu}/precise (100%)
 rename scripts/{ => ubuntu}/quantal (100%)
 rename scripts/{ => ubuntu}/raring (100%)
 rename scripts/{ => ubuntu}/saucy (100%)
 rename scripts/{ => ubuntu}/trusty (100%)
 rename scripts/{ => ubuntu}/utopic (100%)
 rename scripts/{ => ubuntu}/vivid (100%)
 rename scripts/{ => ubuntu}/warty (100%)
 rename scripts/{ => ubuntu}/warty.buildd (100%)
 rename scripts/{ => ubuntu}/wily (100%)
 rename scripts/{ => ubuntu}/xenial (100%)
 rename scripts/{ => ubuntu}/yakkety (100%)
 rename scripts/{ => ubuntu}/zesty (100%)

diff --git a/Makefile b/Makefile
index 8516803..89b35c2 100644
--- a/Makefile
+++ b/Makefile
@@ -10,7 +10,7 @@ install:
 	mkdir -p $(DSDIR)/scripts
 	mkdir -p $(DESTDIR)/usr/sbin
 
-	cp -a scripts/* $(DSDIR)/scripts/
+	cp -a scripts/* $(DSDIR)/scripts/*/*
 	install -o root -g root -m 0644 functions $(DSDIR)/
 
 	sed 's/@VERSION@/$(VERSION)/g' debootstrap >$(DESTDIR)/usr/sbin/debootstrap
diff --git a/scripts/bullseye b/scripts/debian/bullseye
similarity index 100%
rename from scripts/bullseye
rename to scripts/debian/bullseye
diff --git a/scripts/buster b/scripts/debian/buster
similarity index 100%
rename from scripts/buster
rename to scripts/debian/buster
diff --git a/scripts/etch b/scripts/debian/etch
similarity index 100%
rename from scripts/etch
rename to scripts/debian/etch
diff --git a/scripts/etch-m68k b/scripts/debian/etch-m68k
similarity index 100%
rename from scripts/etch-m68k
rename to scripts/debian/etch-m68k
diff --git a/scripts/jessie b/scripts/debian/jessie
similarity index 100%
rename from scripts/jessie
rename to scripts/debian/jessie
diff --git a/scripts/jessie-kfreebsd b/scripts/debian/jessie-kfreebsd
similarity index 100%
rename from scripts/jessie-kfreebsd
rename to scripts/debian/jessie-kfreebsd
diff --git a/scripts/lenny b/scripts/debian/lenny
similarity index 100%
rename from scripts/lenny
rename to scripts/debian/lenny
diff --git a/scripts/oldoldstable b/scripts/debian/oldoldstable
similarity index 100%
rename from scripts/oldoldstable
rename to scripts/debian/oldoldstable
diff --git a/scripts/oldstable b/scripts/debian/oldstable
similarity index 100%
rename from scripts/oldstable
rename to scripts/debian/oldstable
diff --git a/scripts/potato b/scripts/debian/potato
similarity index 100%
rename from scripts/potato
rename to scripts/debian/potato
diff --git a/scripts/sarge b/scripts/debian/sarge
similarity index 100%
rename from scripts/sarge
rename to scripts/debian/sarge
diff --git a/scripts/sarge.buildd b/scripts/debian/sarge.buildd
similarity index 100%
rename from scripts/sarge.buildd
rename to scripts/debian/sarge.buildd
diff --git a/scripts/sarge.fakechroot b/scripts/debian/sarge.fakechroot
similarity index 100%
rename from scripts/sarge.fakechroot
rename to scripts/debian/sarge.fakechroot
diff --git a/scripts/sid b/scripts/debian/sid
similarity index 100%
rename from scripts/sid
rename to scripts/debian/sid
diff --git a/scripts/squeeze b/scripts/debian/squeeze
similarity index 100%
rename from scripts/squeeze
rename to scripts/debian/squeeze
diff --git a/scripts/stable b/scripts/debian/stable
similarity index 100%
rename from scripts/stable
rename to scripts/debian/stable
diff --git a/scripts/stretch b/scripts/debian/stretch
similarity index 100%
rename from scripts/stretch
rename to scripts/debian/stretch
diff --git a/scripts/testing b/scripts/debian/testing
similarity index 100%
rename from scripts/testing
rename to scripts/debian/testing
diff --git a/scripts/unstable b/scripts/debian/unstable
similarity index 100%
rename from scripts/unstable
rename to scripts/debian/unstable
diff --git a/scripts/wheezy b/scripts/debian/wheezy
similarity index 100%
rename from scripts/wheezy
rename to scripts/debian/wheezy
diff --git a/scripts/woody b/scripts/debian/woody
similarity index 100%
rename from scripts/woody
rename to scripts/debian/woody
diff --git a/scripts/woody.buildd b/scripts/debian/woody.buildd
similarity index 100%
rename from scripts/woody.buildd
rename to scripts/debian/woody.buildd
diff --git a/scripts/aequorea b/scripts/tanglu/aequorea
similarity index 100%
rename from scripts/aequorea
rename to scripts/tanglu/aequorea
diff --git a/scripts/bartholomea b/scripts/tanglu/bartholomea
similarity index 100%
rename from scripts/bartholomea
rename to scripts/tanglu/bartholomea
diff --git a/scripts/chromodoris b/scripts/tanglu/chromodoris
similarity index 100%
rename from scripts/chromodoris
rename to scripts/tanglu/chromodoris
diff --git a/scripts/dasyatis b/scripts/tanglu/dasyatis
similarity index 100%
rename from scripts/dasyatis
rename to scripts/tanglu/dasyatis
diff --git a/scripts/artful b/scripts/ubuntu/artful
similarity index 100%
rename from scripts/artful
rename to scripts/ubuntu/artful
diff --git a/scripts/bionic b/scripts/ubuntu/bionic
similarity index 100%
rename from scripts/bionic
rename to scripts/ubuntu/bionic
diff --git a/scripts/breezy b/scripts/ubuntu/breezy
similarity index 100%
rename from scripts/breezy
rename to scripts/ubuntu/breezy
diff --git a/scripts/dapper b/scripts/ubuntu/dapper
similarity index 100%
rename from scripts/dapper
rename to scripts/ubuntu/dapper
diff --git a/scripts/edgy b/scripts/ubuntu/edgy
similarity index 100%
rename from scripts/edgy
rename to scripts/ubuntu/edgy
diff --git a/scripts/feisty b/scripts/ubuntu/feisty
similarity index 100%
rename from scripts/feisty
rename to scripts/ubuntu/feisty
diff --git a/scripts/gutsy b/scripts/ubuntu/gutsy
similarity index 100%
rename from scripts/gutsy
rename to scripts/ubuntu/gutsy
diff --git a/scripts/hardy b/scripts/ubuntu/hardy
similarity index 100%
rename from scripts/hardy
rename to scripts/ubuntu/hardy
diff --git a/scripts/hoary b/scripts/ubuntu/hoary
similarity index 100%
rename from scripts/hoary
rename to scripts/ubuntu/hoary
diff --git a/scripts/hoary.buildd b/scripts/ubuntu/hoary.buildd
similarity index 100%
rename from scripts/hoary.buildd
rename to scripts/ubuntu/hoary.buildd
diff --git a/scripts/intrepid b/scripts/ubuntu/intrepid
similarity index 100%
rename from scripts/intrepid
rename to scripts/ubuntu/intrepid
diff --git a/scripts/jaunty b/scripts/ubuntu/jaunty
similarity index 100%
rename from scripts/jaunty
rename to scripts/ubuntu/jaunty
diff --git a/scripts/karmic b/scripts/ubuntu/karmic
similarity index 100%
rename from scripts/karmic
rename to scripts/ubuntu/karmic
diff --git a/scripts/lucid b/scripts/ubuntu/lucid
similarity index 100%
rename from scripts/lucid
rename to scripts/ubuntu/lucid
diff --git a/scripts/maverick b/scripts/ubuntu/maverick
similarity index 100%
rename from scripts/maverick
rename to scripts/ubuntu/maverick
diff --git a/scripts/natty b/scripts/ubuntu/natty
similarity index 100%
rename from scripts/natty
rename to scripts/ubuntu/natty
diff --git a/scripts/oneiric b/scripts/ubuntu/oneiric
similarity index 100%
rename from scripts/oneiric
rename to scripts/ubuntu/oneiric
diff --git a/scripts/precise b/scripts/ubuntu/precise
similarity index 100%
rename from scripts/precise
rename to scripts/ubuntu/precise
diff --git a/scripts/quantal b/scripts/ubuntu/quantal
similarity index 100%
rename from scripts/quantal
rename to scripts/ubuntu/quantal
diff --git a/scripts/raring b/scripts/ubuntu/raring
similarity index 100%
rename from scripts/raring
rename to scripts/ubuntu/raring
diff --git a/scripts/saucy b/scripts/ubuntu/saucy
similarity index 100%
rename from scripts/saucy
rename to scripts/ubuntu/saucy
diff --git a/scripts/trusty b/scripts/ubuntu/trusty
similarity index 100%
rename from scripts/trusty
rename to scripts/ubuntu/trusty
diff --git a/scripts/utopic b/scripts/ubuntu/utopic
similarity index 100%
rename from scripts/utopic
rename to scripts/ubuntu/utopic
diff --git a/scripts/vivid b/scripts/ubuntu/vivid
similarity index 100%
rename from scripts/vivid
rename to scripts/ubuntu/vivid
diff --git a/scripts/warty b/scripts/ubuntu/warty
similarity index 100%
rename from scripts/warty
rename to scripts/ubuntu/warty
diff --git a/scripts/warty.buildd b/scripts/ubuntu/warty.buildd
similarity index 100%
rename from scripts/warty.buildd
rename to scripts/ubuntu/warty.buildd
diff --git a/scripts/wily b/scripts/ubuntu/wily
similarity index 100%
rename from scripts/wily
rename to scripts/ubuntu/wily
diff --git a/scripts/xenial b/scripts/ubuntu/xenial
similarity index 100%
rename from scripts/xenial
rename to scripts/ubuntu/xenial
diff --git a/scripts/yakkety b/scripts/ubuntu/yakkety
similarity index 100%
rename from scripts/yakkety
rename to scripts/ubuntu/yakkety
diff --git a/scripts/zesty b/scripts/ubuntu/zesty
similarity index 100%
rename from scripts/zesty
rename to scripts/ubuntu/zesty
-- 
2.16.2

